### DISCUSSION TITLE
Hot reload for razor views in custom theme using VS 2022

### DISCUSSION THREAD

**Discussion by gjuszczak** [OrchardCMS/OrchardCore#15321] [Q&A]
Hot reloading of razor views located in the theme is not working. I am using OC 1.8.2 and VS 2022. The hot reloading log shows that it was completed successfully. The web-browser refreshes itself but the changes are not applied. It is working perfectly fine though for liquid templates and also when I switch to the previous OC 1.7.2 (net 7.0) version. 

Are you also facing this issue?

**Comment by aaronamm** [comment]
@gjuszczak  I have the same issued after updated to 1.8.3 and .NET 8.
Not sure if you find the solution.

**Reply by aaronamm** [reply]
@gjuszczak  confirmed.
Rolling back to OC 1.7.2 + NET 8 fixes the issue.

**Comment by aaronamm** [comment]
I have opened an issue for this problem. #15946
